apps/web/src/hooks/usePermissionConstants.ts

```diff
@@ -0,0 +1,34 @@
+import { RoleName } from "@gc-digital-talent/auth";
+
+const usePermissionConstants = () => {
```

Comment:
Actually, I don't think this should be a "hook". Putting code in a "hook" allows it to use some React magic, like other hooks, but comes with a downside: hooks can only be used inside React components. This is a simple set of constants, it doesn't reference any other hooks, so there's no reason to call it a hook and restrict it to React components.

I would move/rename this file to `apps/web/src/constants/permissionConstants.ts`

Reply:
Sounds good, fixed in [654171e](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11501/commits/654171e44414d2061a017efb530ea695de63d4b9)